app/models/establishment.rb

```diff
@@ -30,9 +30,15 @@ def directors
     "code_postal" => :postal_code,
     "nom_commune" => :city,
     "telephone" => :telephone,
-    "mail" => :email
+    "mail" => :email,
+    "code_type_contrat_prive" => :private_contract_type_code
   }.freeze
 
+  # https://infocentre.pleiade.education.fr/bcn/workspace/viewTable/n/N_CONTRAT_ETABLISSEMENT
+  STATUS_CONTRACTS_PUBLIC = ["99"].freeze
+  STATUS_CONTRACTS_PRIVATE_ALLOWED = %w[30 31 40 41].freeze
+  STATUS_CONTRACTS_OTHER = %w[10 20 21 ** !! 50].freeze
```

Comment:
I like it 😎 

pas besoin de stocker les other, si ?

et sur le nommage je trouve ça plus naturel `CONTRACT_STATUS_PUBLIC` `CONTRACT_STATUS_PRIVATE_ALLOWED` mais à ta guise

Reply:
J'ai changé ça avant de voir tes comms